test/library/standard/List/perf/listBenchmark1.compopts

```diff
@@ -0,0 +1 @@
+--fast
```

Comment:
We usually don't throw --fast for correctness testing. If the program takes too long to run, you could just lower the default problem size.

Reply:
Should I just go ahead and add that to the `.perfcompopts` instead?

Reply:
start_test will automatically compile with --fast with '-performance' thrown, so no need.

Reply:
Thanks! I didn't know that.